### PR TITLE
Added "Comparing arrays and strings" section

### PR DESCRIPTION
**Pull request by codeAbinash**
#### Summary
Added "Comparing Dates and strings" section in the "Equality Operator(==)" page

#### Motivation
"Comparing Dates and strings" is needed in the "[Equality Operator(==)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality)" page. It is part of the topic. 

#### Supporting details

ECMAScript Specification : [Array.prototype.toString()](https://262.ecma-international.org/5.1/#sec-15.4.4.2), [Object.prototype.toString()](https://262.ecma-international.org/5.1/#sec-15.2.4.2)

#### Metadata
- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/operators/equality/index.md [hunk 1]
```diff
@@ -127,6 +127,18 @@ const s = d.toString(); // for example: "Sun Dec 17 1995 03:24:00 GMT-0800 (Paci
 console.log(d == s);    //true
 ```
 
+### Comparing arrays and strings
```
Comment (codeAbinash) [inline files/en-us/web/javascript/reference/operators/equality/index.md:130]: Added "Comparing arrays and strings" section

File: files/en-us/web/javascript/reference/operators/equality/index.md [hunk 2]
```diff
@@ -127,6 +127,18 @@ const s = d.toString(); // for example: "Sun Dec 17 1995 03:24:00 GMT-0800 (Paci
...
+
+```js
```
Comment (codeAbinash) [inline files/en-us/web/javascript/reference/operators/equality/index.md:132]: Examples of comparing arrays and strings

**Review by codeAbinash — COMMENTED** [review]
Added "Comparing arrays and strings" section

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**